.github/workflows/release-publish-suite-commit.yml

```diff
@@ -0,0 +1,66 @@
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  release-tests:
```

Comment:
can we rename the existing job to `draft-release` instead of `release` so that it's clear how these workflows are different? maybe also rename `test.yml` to `suite-unit-tests.yml` and rename `release-test.yml` to whatever it actually is (running the e2e tests of the suite against the latest versions? other?)